Floofbot/Services/Repository/Models/Tag.cs

```diff
@@ -6,7 +6,8 @@ public partial class Tag
     {
         [Key]
         public ulong TagId { get; set; }
+        public string TagName { get; set; }
         public ulong UserId { get; set; }
```

Comment:
what's the purpose of user id? I think server id should be logged somewhere too, so that tags are not global

Reply:
Added a ServerId field.
I kept the UserId field so it stays clear who added what.